docs/reference/data-type-support.rst

```diff
@@ -0,0 +1,144 @@
+.. meta::
+   :description: hipBLASLt library data type support
+   :keywords: hipBLASLt, ROCm, data type support
+
+.. _data-type-support:
+
+******************************************
+Data type support
+******************************************
+
+This topic lists the supported data types for the hipBLASLt GEMM operation, 
+which is performed by :ref:`hipblasltmatmul`. Here is the equation:
+
+.. math::
+
+ D = Activation(alpha \cdot op(A) \cdot op(B) + beta \cdot op(C) + bias)
+
+Where :math:`op( )` refers to in-place operations, such as transpose and
+non-transpose, and :math:`alpha` and :math:`beta` are scalars.
+
+.. note:: 
+  
+  The ``__hip_fp8_e4m3_fnuz`` and ``__hip_fp8_e5m2_fnuz`` data types are only
```

Comment:
The actual data type used by user are hipblaslt_f8_fnuz and hipblaslt_bf8_fnuz.
Any reason that you use __hip_fp8_e4m3_fnuz and __hip_fp8_e5m2_fnuz?
And should we have a table to match data type to HIP enum data type? example: HIP_R_8F_E4M3_FNUZ

Reply:
@jichangjichang I added the compute modes and hipBLASLt type list. It can be, that I shared modes, which are not supported, please check it. 

Also removed the HIP types, and now I am adding the HIP enum data type map to the main data type support page.